### DISCUSSION TITLE
Thank you for this rad "continuous thing-doer"!

### DISCUSSION THREAD

**Discussion by markmark206** [concourse/concourse#6165] [Show and tell]
Hi!

I would like to thank the Concourse team for building such a rad  "continuous thing-doer!"

[Our tiny startup](https://temporal.io/) just announced a round of funding, and the front page of [SD Times](https://sdtimes.com/) is currently prominently featuring a screenshot of our Concourse CI installation, running our test and release automation pipelines. ;)

![image](https://user-images.githubusercontent.com/218696/96205906-8e8c6a80-0f1c-11eb-9693-5ab62b1d00b9.png)

Funny enough, one of the jobs is failing – I guess that's how you know it's the real deal. 😂😂

Thank you for such a great tool!
Mark.
PS [Direct link to the article](https://sdtimes.com/micro/temporal-raises-18-75-million-for-microservice-orchestration/).

**Comment by vito** [comment]
Haha, hey that's awesome. Thanks for sharing, and congrats!

I guess Concourse is a good fallback for when you don't have more topical clipart for project updates. 😆

**Comment by rylandg** [comment]
Hey @vito, I'm the head of product for Temporal. We have been using Concourse for a while internally and keep theorizing that Temporal might be a great fit for Concourse itself. I would love to chat sometime and see if there is potential overlap.

**Reply by vito** [reply]
Not sure what that means yet haha but happy to chat about it! Feel free to email me if it's private - my address is in my GitHub profile.

**Reply by rylandg** [reply]
Actually not private at all, just much easier to explain on a call. I reached out!
